<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Billpay</class>
 <widget class="QDialog" name="Billpay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>530</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>760</width>
    <height>530</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Georgia</family>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>730</width>
     <height>500</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>730</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>730</width>
     <height>500</height>
    </size>
   </property>
   <property name="title">
    <string>Bill Payment</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>341</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Utility Bill Payment</string>
    </property>
    <widget class="QComboBox" name="comboBox_Utility">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Utility">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 127);</string>
     </property>
     <property name="text">
      <string>PAY NOW</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>81</y>
       <width>121</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Georgia</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 127);</string>
     </property>
     <property name="text">
      <string>Meter Number</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_meterNo">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>81</y>
       <width>121</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Georgia</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Utility">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>251</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>270</y>
      <width>331</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Car Bill Payment</string>
    </property>
    <widget class="QComboBox" name="comboBox_Car">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Choose your bill</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MTV</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_Car">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 127);</string>
     </property>
     <property name="text">
      <string>PAY NOW</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>341</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Mobil Bill Payment</string>
    </property>
    <widget class="QComboBox" name="comboBox_Mobil">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Choose your phone company</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Turkcell</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vodefone</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Turk Telekom</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_Mobil">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 127);</string>
     </property>
     <property name="text">
      <string>PAY NOW</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>331</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Internet Bill Payment</string>
    </property>
    <widget class="QComboBox" name="comboBox_Internet">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Internet">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 127);</string>
     </property>
     <property name="text">
      <string>PAY NOW</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_SubNo">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>121</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Georgia</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>80</y>
       <width>121</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Georgia</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 127);</string>
     </property>
     <property name="text">
      <string>Subs Number</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
